src/main/java/com/minecolonies/coremod/inventory/GuiHandler.java

```diff
@@ -57,7 +60,15 @@ else if (id == ID.CITIZEN_INVENTORY.ordinal())
                                                               building.getID(),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "building" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2259)